lib/cadet/accounts/user.ex

```diff
@@ -27,4 +27,9 @@ defmodule Cadet.Accounts.User do
     |> validate_required(@required_fields)
     |> validate_inclusion(:role, Role.__valid_values__())
   end
+
+  def student_changeset(student, group_id) do
+    student
+    |> change(%{group_id: group_id})
```

Comment:
Erm no. Do not use `change`.

Please take a look at https://github.com/source-academy/cadet/blob/master/lib/cadet/helpers/model_helper.ex; check out `add_belongs_to_id_from_model/`3